Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM golang:1.6
 
+RUN apt-get update
```

Comment:
We need to combine this statement with the install per [Maynard's investigation](https://distil.slack.com/archives/GDVS9S2PR/p1552609394040200).